test/interp/dynamic/mem_dynamic_dispatch.bril

```diff
@@ -24,8 +24,8 @@
     ret;
 
 .call_my_print_2:
-    data : any = load args;
-    call @my_print_2 data;
+    data2 : any = load args;
+    call @my_print_2 data2;
```

Comment:
Good catch… not entirely sure why the `brilck` checks didn't notice this type conflict with `data`. I'll look into this momentarily.

Reply:
Resolved here! Sorry that this will create a merge conflict, probably… https://github.com/sampsyo/bril/pull/422